gulpfile.js

```diff
@@ -20,7 +22,9 @@ require('./').gulpDoxx(require('gulp'),doxxOptions);
 
 
 var ghPagesOptions = {
-
+  config:{
+    remoteUrl: 'git@github.com:pattern-library/pattern-library-utilities.git'
+  },
   src: ['./docs/**/*'],
```

Comment:
@scottnath Does this src directory exist? I don't see a docs directory at the process.cwd() dir of the gulpfile.